.storybook/stories/button/button-group.stories.ts

```diff
@@ -16,7 +16,7 @@ const defaultStory: Story = args => ({
       <clr-button-group [clrMenuPosition]="clrMenuPosition" [clrToggleButtonAriaLabel]="clrToggleButtonAriaLabel">
         <clr-button
           *ngFor="let _ of createArray(buttonCount); let i = index"
-          [clrInMenu]="false"
+          [clrInMenu]="false" [disabled]="disabledButtonsPosition.split(',').includes(i+1+'')"
```

Comment:
```suggestion
          [clrInMenu]="false"
          [disabled]="disabledButtonsPosition.split(',').includes(i+1+'')"
```

nit: This should be on the next line actually (based on how `prettier` formats templates).